eth_account/_utils/transactions.py

```diff
@@ -34,12 +31,22 @@
 
 from .validation import (
     is_valid_address,
+    is_empty_or_checksum_address,
+    is_int_or_prefixed_hexstr,
+    TRANSACTION_FORMATTERS,
+    TRANSACTION_VALID_VALUES,
 )
 
-VALID_EMPTY_ADDRESSES = {None, b'', ''}
+from .typed_transactions import (
+    TypedTransaction
+)
 
 
 def serializable_unsigned_transaction_from_dict(transaction_dict):
+    if 'type' in transaction_dict:
```

Comment:
Here you probably want to avoid entering the condition if there is a transaction type set, but it's 0?

Reply:
the TypedTransaction.from_dict() method handles this behaviour. do you feel it's worth adding another check here?